scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -35,6 +35,19 @@ test-dependency-rules:
   script:
     - ./scripts/ci/gitlab/ensure-deps.sh
 
+test-rust-features:
+  stage:                           check
+  extends:
+    - .kubernetes-env
+    - .test-refs-no-trigger-prs-only
+  allow_failure:                   true
```

Comment:
I marked it as allowed to fail for now, until it is prove not to hold up development.